cmd/node/flags.go

```diff
@@ -351,6 +352,13 @@ var (
 		Usage: "The `filepath` for the PEM file which contains the secret keys for the p2p key. If this is not specified a new key will be generated (internally) by default.",
 		Value: "./config/p2pKey.pem",
 	}
+
+	// operationMode defines the flag for specifying how configs should be altered depending on the node's intent
+	operationMode = cli.StringFlag{
+		Name:  "operation-mode",
+		Usage: "String flag for specifying the desired `operation mode`(s) of the node, resulting in altering some configuration values accordingly. Values are: import-db, full-archive, db-lookup-extension, historical-balances or `\"\"` (empty). Multiple values can be separated via ,",
```

Comment:
maybe add the operation mode: "observer" in which we have the `no-key` flag set?

Reply:
I thought about it, but if I do so, the node operators won't be able to use their preferred key. There aren't other config flags that should be changed, so I didn't include this mode. Can come in the future

Reply:
ok